src/main/java/frc/mechtechsupport/util/SwerveModuleMath.java

```diff
@@ -0,0 +1,69 @@
+package frc.mechtechsupport.util;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.util.Units;
+
+public class SwerveModuleMath {
+    public static double restrictAngle(double angle) {
+        while (angle > 360) {
+            angle -= 360;
+        }
+        while (angle < 0) {
+            angle += 360;
+        }
+        return angle;
+    }
+
+    public static final SwerveDriveKinematics swerveMagik = new SwerveDriveKinematics(
+            new Translation2d(Units.inchesToMeters(24) / 2, Units.inchesToMeters(24) / 2), //FR
+            new Translation2d(Units.inchesToMeters(24) / 2, -Units.inchesToMeters(24) / 2), //FL
+            new Translation2d(-Units.inchesToMeters(24) / 2, Units.inchesToMeters(24) / 2), //BR
+            new Translation2d(-Units.inchesToMeters(24) / 2, -Units.inchesToMeters(24) / 2) //BL
```

Comment:
need to fix this because I incorrectly assumed that negative is left and positive is right, FR (the first one) should be new Translation2d(Units.inchesToMeters(24) / 2, -Units.inchesToMeters(24) / 2), to fix this you basically just need to flip the signs of all of the second inputs